Tiltfile

```diff
@@ -385,6 +395,9 @@ if wormchain:
     guardian_resource_deps = guardian_resource_deps + ["wormchain", "wormchain-deploy"]
 if sui:
     guardian_resource_deps = guardian_resource_deps + ["sui"]
+# TON watcher doesn't need a separate devnet resource - it connects to real TON testnet
+# if ton:
+#     guardian_resource_deps = guardian_resource_deps + ["ton"]
```

Comment:
Delete unused code.
We'll make a readme file with instructions how to setup ton in local dev environment. Let's move this comment to that readme.